portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,154 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <div id = "banner">
+        <h1>Hello World!</h1>
+        <div id="banner-content">Cirque de Towers, Wyoming</div>
+    </div>    
+
+    <div id = "profile-pic">
+        <img style = "width: 200px; height = 200px" src="/images/siteimages/climbing.jpg">
```

Comment:
The syntax for css is incorrect. It should be "height: 200px". Also, let's try to make our code consistent in styling, e.g. whether to have 'space' before and after '='. I see you using 'id = "xxx"', and 'src="xxx"'. The recommended style should be WITHOUT space for all the html attribute. Same for all the other change in this file

Reply:
Syntax errors fixed, formatting standardized